src/compiler/checker.ts

```diff
@@ -1542,6 +1542,8 @@ export function createTypeChecker(host: TypeCheckerHost): TypeChecker {
         onSuccessfullyResolvedSymbol,
     });
 
+    var inFullCheckAfterPartial = false;

```

Comment:
This seems redundant with the `NodeFlags.PartiallyTypeChecked` flag itself. Isn't checking that node flag sufficient?